src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -2841,6 +2841,7 @@ def set_vmss_extension(cmd, resource_group_name, vmss_name, extension_name, publ
     ext = VirtualMachineScaleSetExtension(name=extension_instance_name,
                                           publisher=publisher,
                                           type=extension_name,
```

Comment:
seems VirtualMachineScaleSetExtension constructor do not have `type` anymore, why not delete it directly ?

Reply:
agree